zoomus/components/webinar.py

```diff
@@ -18,6 +18,12 @@ def list(self, **kwargs):
             kwargs['start_time'] = util.date_to_str(kwargs['start_time'])
         return self.post_request("/webinar/list", params=kwargs)
 
+    def upcoming(self, **kwargs):
```

Comment:
Can you add a test file tests/zoomus/components/webinar/test_upcoming.py and add in the appropriate tests. Take a look at the other files there for examples. Thanks!